feature/eks-cluster-creation/main.tf

```diff
@@ -2,6 +2,10 @@
 # # https://learn.hashicorp.com/terraform/kubernetes/provision-eks-cluster#optional-configure-terraform-kubernetes-provider
 # # To learn how to schedule deployments and services using the provider, go here: ttps://learn.hashicorp.com/terraform/kubernetes/deploy-nginx-kubernetes.
 
+terraform {
+  required_version = ">= 0.12"
+}
+
```

Comment:
I had the same in my AKS but I change the value to "=0.12.6" in order to pin exacly one version. Maybe it could be good standard. But still it is small detail.

Reply:
I think, I need to keep it as it is, as when I try to put this version, I get:
`Error: Unsupported Terraform Core version
  on main.tf line 6, in terraform:
   6: 	required_version = "=0.12.6"`

Reply:
Yes exacly, since terraform installed the latest version i suppose and all the time there will be installed the latest version using this code, we don't even know which one exacly. So maybe pin to the latest version for example 0.13.0? Using >= is similar to using docker images without any tag which is danger in some cases in my opinion.